### DISCUSSION TITLE
New Templates 4.2.0

### DISCUSSION THREAD

**Discussion by JanOlsmar** [microsoft/fluentui-blazor#1115] [Q&A] [answered]
When I install templates and follow instructions here:
https://www.nuget.org/packages/Microsoft.FluentUI.AspNetCore.Templates/
It seems to me I get wrong templates installed.

I get this:
<Project Sdk="Microsoft.NET.Sdk.Web">

	<PropertyGroup>
		<TargetFrameworks>net6.0;net7.0</TargetFrameworks>
		<Nullable>enable</Nullable>
		<ImplicitUsings>enable</ImplicitUsings>
		<IsPackable>true</IsPackable>
	</PropertyGroup>

	<ItemGroup>
		<PackageReference Include="Microsoft.Fast.Components.FluentUI" Version="2.*" />
	</ItemGroup>

</Project>

Same behavior for server and wasm project.

Thanks in advance :)

**Comment by vnbaaij** [comment]
You probably have the older (v3) templates installed as well. 

What are the actions you have done to get the code in the project file you are showing?

**Reply by JanOlsmar** [reply]
1. dotnet new uninstall Microsoft.FluentUI.AspNetCore.Templates
2. dotnet new install Microsoft.FluentUI.AspNetCore.Templates::4.2.0
3. dotnet new fluentuiblazorserver -o  xxx
4. And opened a new project VS same behavior.
5. Did the same for wasm

**Reply by vnbaaij** [reply, marked as answer]
`fluentuiblazorserver` calls creating a site with the v3 template...
For the new templates use `fluentblazor` or `fluentblazorwasm`

I see we have used the olde ones in the readme. Sorry about that! Will fix asap.

**Reply by JanOlsmar** [reply]
I did install new machine and find out the name error. All worked fine there after I changed name.
But why do I get wrong in VisualStudio on the first machine.
I fix it some how.
Thanks for fast answer.
